core-bundle/src/Command/SymlinksCommand.php

```diff
@@ -91,15 +91,23 @@ public function __construct(string $projectDir, string $uploadPath, string $logs
     protected function configure(): void
     {
         $this
-            ->addArgument('target', InputArgument::OPTIONAL, 'The target directory', 'web')
-            ->setDescription('Symlinks the public resources into the web directory.')
+            ->addArgument('target', InputArgument::OPTIONAL, 'The target directory')
+            ->setDescription('Symlinks the public resources into the public directory.')
         ;
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $this->webDir = $input->getArgument('target');
 
+        if (null === $this->webDir) {
+            if ((new Filesystem())->exists($this->projectDir.'/web')) {
+                $this->webDir = 'web'; // backwards compatibility
+            } else {
+                $this->webDir = 'public';
+            }
+        }
```

Comment:
Now testing this, I'm wondering why we do not just rely on the config value here? IMHO the config should be the single point of truth once configured.